src/api/jobs.js

```diff
@@ -16,16 +16,16 @@ export async function getRunningJobs(accessionCode) {
   const [processorJobs, downloaderJobs] = await Promise.all([
     Ajax.get(`/v1/jobs/downloader/`, {
       sample_accession_code: accessionCode,
-      nomad: true,
+      is_queued: true,
     }),
     Ajax.get(`/v1/jobs/processor/`, {
       sample_accession_code: accessionCode,
-      nomad: true,
+      is_queued: true,
```

Comment:
This doesn't work. It's no longer a filter and instead is just a property that is added on every result. This is because we'd have to make potentially multiple API calls to Batch to get all the currently active job ids, then pass that to the queryset as a filter. That could make it quite slow, and I think just having that as a field on the results should be sufficient right?